lms/djangoapps/lti_provider/views.py

```diff
@@ -70,11 +83,13 @@ def lti_launch(request, course_id, usage_id):
             params['oauth_consumer_key']
         )
     except LtiConsumer.DoesNotExist:
-        return HttpResponseForbidden()
+        template403 = Template(render_to_string('static_templates/403.html', {'return_url': return_url}))
```

Comment:
Will be good to avoid duplication. You may create the separate method - something like `render_response_forbidden` with the content:

```
def render_response_forbidden(self, return_url):
    template403 = Template(render_to_string('static_templates/403.html', {'return_url': return_url}))
    return HttpResponseForbidden(template403.render())